compiler/pkg/generator/template/graphql/graphqlResolver.go.tmpl

```diff
@@ -65,6 +65,8 @@ func getRootResolver() (*model.{{$node.PkgName}}{{$node.NodeName}}, error) {
             return nil, fmt.Errorf("failed to get k8s client config: %s", err)
 	    }
 	nc = nexusClient
+	nc.SubscribeAll()
```

Comment:
@LokeshvarKr You will need this in the non singleton root resolver in the else{} case below.

Reply:
Thanks @xmen4xp , I have updated the code PTAL